cmd/config.go

```diff
@@ -271,5 +272,40 @@ func RunConfigV1() {
 			fmt.Printf("Error: %s\n", err)
 			os.Exit(1)
 		}
+		newWatcher, err := fsnotify.NewWatcher()
+		if err != nil {
+			fmt.Printf("Error: %s\n", err)
+			os.Exit(1)
+		}
+		defer newWatcher.Close()
+		fmt.Println("Watching for changes to config file:", configPath)
+		err = newWatcher.Add(configPath)
+		if err != nil {
+			fmt.Printf("Error: %s\n", err)
+			os.Exit(1)
+		}
+		go func() {
+			for {
+				select {
+				case event, ok := <-newWatcher.Events:
+					if !ok {
+						return
+					}
+					if event.Op&fsnotify.Write == fsnotify.Write {
+						fmt.Printf("Config file changed: %s file: %s", event.Name, configPath)
+						err = controlsvc.InitReload(configPath, reloadParseAndRun)
+						if err != nil {
+							fmt.Printf("Error: %s\n", err)
+							os.Exit(1)
+						}
+					}
+				case err, ok := <-newWatcher.Errors:
+					if !ok {
+						return
+					}
+					fmt.Println("Error:", err)
+				}
+			}
+		}()
```

Comment:
We need a done chan to block the main goroutine from exiting.

`done := make(chan bool)`

and on line 310:
`<-done`